packages/manager/modules/sms/src/sms/batches/history/delete/delete.controller.js

```diff
@@ -0,0 +1,26 @@
+export default class TelecomSMSBatchesHistoryDeleteCtrl {
+  /* @ngInject */
+  constructor(
+    $http,
+    // Alerter,
+    $translate,
+  ) {
+    this.isLoading = false;
+    this.$http = $http;
+    // this.Alerter = Alerter;
```

Comment:
If no longer need, you can delete code in comments

Reply:
Comment on a file that should be part of the release, sorry!